index.tsx

```diff
@@ -58,8 +58,8 @@ const cacheThing = findByPropsLazy("commit", "getOrCreate");
 
 const handledMessageIds = new Set();
 async function messageDeleteHandler(payload: MessageDeletePayload & { isBulk: boolean; }) {
-    if (payload.mlDeleted && payload.id === "ml-remove-history") {
-        if (settings.store.permanentlyRemoveLogByDefault) removeLog(payload.id);
+    if (payload.mlDeleted) {
+        if (payload.id === "ml-remove-history" && settings.store.permanentlyRemoveLogByDefault) removeLog(payload.id);
```

Comment:
"ml-remove-history" is a id for their context menu and doenst get dispatched, so that would have always been false.
so i just removed that check

https://github.com/Vendicated/Vencord/blob/main/src/plugins/messageLogger/index.tsx#L46
![image](https://github.com/Syncxv/vc-message-logger-enhanced/assets/47534062/fd154774-62a0-43b3-aced-7206de596203)